### PR TITLE
Remove constants repetition and duplicate define

### PR DESCRIPTION
**Pull request by Starbuck5**
I noticed that constants compiled into one of the largest of the pygame modules, so I was looking around constants. I figured that we could use the `__all__` list generated by constants.c instead of redefining the whole thing in locals.py. While validating that my changes didn't affect any output, I discovered that `GL_ACCELERATED_VISUAL` is added twice in constants.c by mistake (the same way each time), so I took out a duplicate definition.

### CODE REVIEW

**Review by ankith26 — APPROVED** [review]
Nice simplifications! Thanks 👍 🎉

**Review by zoldalma999 — APPROVED** [review]
Looks good to me! 🎉

File: src_py/locals.py [hunk 1]
```diff
@@ -19,573 +19,12 @@
 #    pete@shinners.org
 
 
-"""Set of functions from PyGame that are handy to have in
+"""Set of functions from pygame that are handy to have in
```
Comment (illume) [inline src_py/locals.py:22]: lowercase pygame makes me feel happy

**Review by illume — COMMENTED** [review]
Indeed, nice deletion! :)

**Pull request merged**